worker/src/main/java/gov/cms/ab2d/worker/processor/OutputHelper.java

```diff
@@ -0,0 +1,34 @@
+package gov.cms.ab2d.worker.processor;
+
+import gov.cms.ab2d.aggregator.FileOutputType;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.codec.binary.Hex;
+import org.apache.commons.codec.digest.DigestUtils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+/**
+ * Contains the common methods for other StreamHelper implementations
+ */
+@Slf4j
+@SuppressWarnings("checkstyle:visibilitymodifier")
+public class OutputHelper {
```

Comment:
This is only used in one place and doesn't justify being in its own class.  Simply inline the method into ContractProcesserImpl.

Static classes and static methods in general should be avoided.

Reply:
It was a severely downsized version of code that existed before in the text writer that AB2D no longer has. I can refactor.